src/main/java/com/checkmarx/flow/config/RepoProperties.java

```diff
@@ -103,6 +103,9 @@ public void setBlockMerge(boolean blockMerge) {
         this.blockMerge = blockMerge;
     }
 
+    /**
+     * Indicates if threshold checking functionality is enabled.
+     */
     public boolean isErrorMerge() {
         return errorMerge;
```

Comment:
Add Annotation

Reply:
I couldn't find an appropriate annotation in Spring.